# sh/.diff = Compare the last backup made by .cp, against the present

if [ $# = 0 ]; then
    P=$(find . -maxdepth 1 -not -type d -print0 |xargs -0 -r ls -rt |tail -1)
    if [ "$P" ]; then
       set "$P"
    fi
fi

set -xe
diff -brpu $(ls -rt "$@"~*~) "$@"
